src/Features/Request.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Features;
+
+use Behat\Behat\Context\Context;
+use Blumilk\BLT\Features\Traits\HttpRequest as HttpRequestTrait;
+
+class Request implements Context
+{
+    use HttpRequestTrait;
+}
```

Comment:
You're loosing unified approach with naming here. Every trait and class is named the same way, but here it's `Request` and `HttpRequest`. Please unify it to avoid confusion.